ocs_ci/utility/utils.py

```diff
@@ -1137,3 +1137,27 @@ def ceph_health_check(namespace=None):
         raise CephHealthException(
             f"Ceph cluster health is not OK. Health: {health}"
         )
+
+
+def get_rook_repo(to_checkout):
+    """
+    Clone and checkout the rook repository to specific branch/commit.
+
+    Args:
+        to_checkout (str): Commit id or branch to checkout on
+
+    """
+    fresh_clone = False
+    cwd = constants.ROOK_REPO_DIR
+    if not os.path.isdir(constants.ROOK_REPO_DIR):
```

Comment:
What is the reason for using `constants.ROOK_REPO_DIR` instead of `cwd`? Wouldn't be `cwd` slightly more clear since it is assigned just line above?

Reply:
done, forgot to change it here, thanks